doc/modules/ROOT/pages/projects.adoc

```diff
@@ -715,6 +715,18 @@ your project, you could customize it with the following:
 ((nil . ((projectile-project-name . "your-project-name-here"))))
 ----
 
+By default, compilation buffers are not writable, which allows you to
+e.g.  press `g` to restart the last command. Setting
+`projectile-<cmd>-use-comint-mode` (where `<cmd>` is `configure`,
+`compile`, `test`, `install`, `package`, or `run`) to a non-nil value
+allows you to make projectile compilation buffers interactive, letting
+you e.g. test a command-line program with `projectile-run-project`.
+
+[source,elisp]
+----
+(setq projectile-comint-mode t)
```

Comment:
You forgot to update this.

Reply:
What do you mean ? I did update it to reflect the new variable names